tauv_common/config/thruster_manager.yaml

```diff
@@ -0,0 +1,50 @@
+thruster_manager:
+  tf_prefix: manta
+  base_link: base_link
+  thruster_topic_prefix: thrusters/
+  thruster_topic_suffix: /input
+  thruster_frame_base: thruster_
```

Comment:
the prefix/suffix stuff here is kind of confusing. Can you add a comment to indicate what the resulting topic tree will look like for the defined prefix, suffix, etc?